modulesets/gstreamer.modules

```diff
@@ -97,6 +97,15 @@
     <branch repo="slomo" module="gst-player" checkoutdir="gst-player"/>
   </autotools>
 
+  <autotools id="gst-editing-services" autogenargs="--disable-gtk-doc">
+    <dependencies>
+      <dep package="libxml2"/>
+      <dep package="gstreamer"/>
+      <dep package="gst-devtools"/>
```

Comment:
it might be better to add all of gstreamer plugins into dependencies because the more plugins the more capcities I suppose. (but gst-devtools might not)